ptn/boozebot/botcommands/Departures.py

```diff
@@ -0,0 +1,292 @@
+"""
+Cog for unloading related commands
+
+"""
+
+# libraries
+import re
+from typing import List
```

Comment:
nitpick: on 3.9 onward you don't need this, `list[str]` is valid annotation

(setup.py says >=3.9, but dockerfile runs 3.10)